Userland/Libraries/LibAudio/Sample.h

```diff
@@ -37,8 +37,8 @@ ALWAYS_INLINE double linear_to_amplitude(double const value)
 {
     // Linear slope at the lower values to avoid asymptotic behaviour
     // NOTE: These values are dependent on the dynamic range and thus the bit depth.
-    if (value < 0.05)
-        return value * 0.028;
+    if (value < 0.1)
+        return value * 0.00045;
```

Comment:
It would be a good idea to put these up with the constants, because as you say they depend on bit depth.